recipes/pyerfa/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pyerfa" %}
+{% set version = "1.7.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 58cd5fbc27e31211102568f952fdbac410b184563be4da57743cbe5e5ccebe47
+
+build:
+  number: 0
+  skip: true  # [py<3.6]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - setuptools_scm
+    - jinja2 >=2.10.3
+    - python
+    - numpy 1.16.*
```

Comment:
Does this need to be pinned so tightly. Wouldn't >=1.16 work?

Reply:
I was honestly a little confused. The minimum supported numpy version is 1.16, so I thought I should build against that one, but the runtime constraint is `>=1.16` and am happy to change it in host too.

Reply:
I think this is fine, just curios